src/features/metadata-instance-editor/MetadataInstanceEditor.js

```
@@ -15,6 +15,7 @@ type Props = {
     blurExceptionClassNames?: Array<string>,
     canAdd: boolean,
     canUseAIFolderExtraction?: boolean,
+    canUseAIFolderExtractionAgentSelector?: Boolean,
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)